### ISSUE TITLE
About New Introduced Version

### ISSUE THREAD

**Issue by Choneke** [ifzhang/FairMOT#385]
@ifzhang , Thank you for introducing the light version to FairMOT. 

But I have some question, on this new version.

1. How many number of epoch you use to train the model?
2. Does using this model with YOLO as a detector, will the method remain the anchor free??

Thank you

**Comment by ifzhang** [comment]
1. We train 30 epochs as normal, but some settings are different, such as learning rate, batchsize and some loss weights. You can find them in experiments/all_yolov5s.sh
2. The detector is also anchor-free. We only change the backbone using yolov5s and add some upsample layers.

**Comment by Choneke** [comment]
Thank you for clarifications.

**Comment by zengjie617789** [comment]
Is there anyone who test the yolov5 on MOT16 ? I used the yolov5s weights shared by the author , but the results in not as good as expected. Here are the results below:
![image-20210628111107672-1624849869111](https://user-images.githubusercontent.com/25050291/124464127-61435d80-ddc6-11eb-9b31-29a53f0bd4cf.png)
the difference between 256 and 128 reid_dim  is trival.

**Comment by zhaoxin111** [comment]
> 1. We train 30 epochs as normal, but some settings are different, such as learning rate, batchsize and some loss weights. You can find them in experiments/all_yolov5s.sh
> 2. The detector is also anchor-free. We only change the backbone using yolov5s and add some upsample layers.

Hi, I found the det result of img from MOT are much better than otther imgs, such as imgs from crowhhuman.
Did you finetuning the pretrained yolo MOT model on MOT dataset?
